.github/workflows/dart_ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dart-lang/setup-dart@v1
         with:
-          sdk: stable
+          sdk: 2.18.7
```

Comment:
since 3.0 got released to stable, it now doesn't work .. so avoiding that for now.